Lib/gflanguages/data/scripts/Dupl.textproto

```diff
@@ -1,3 +1,4 @@
 id: "Dupl"
 name: "Duployan shorthand"
-
+family: "American"
+summary: "Duployan shorthand (Sloan-Duployan shorthand, Duployan stenography) is an European alphabet, written left-to-right. Geometric stenography script created in 1860 by Father Émile Duployé for writing French, later expanded and adapted for writing English, Chinook Jargon and many others. Heavily cursive (connected), allows words to be written in a single stroke. Praised for simplicity and speed of writing. Needs software support for complex text layout (shaping)."
```

Comment:
“an European” should be “a European”.

Reply:
The `family` should be `"European"`.

Reply:
I'll merge this and fix, just because I'm handling a bunch of changes at ones at the moment.